lms/validation/authentication/_bearer_token.py

```diff
@@ -60,6 +60,8 @@ class BearerTokenSchema(PyramidRequestSchema):
     user_id = marshmallow.fields.Str(required=True)
     oauth_consumer_key = marshmallow.fields.Str(required=True)
     roles = marshmallow.fields.Str(required=True)
+    tool_consumer_instance_guid = marshmallow.fields.Str(required=True)
+    display_name = marshmallow.fields.Str(required=True)
```

Comment:
`BearerTokenSchema` can just continue doing what it always did to create an `LTIUser`: just `LTIUser(**data)`. But `data` will now contain the `tool_consumer_instance_guid` and `display_name` fields.